html-aam/img-src-srcset-roles.tentative.html

```diff
@@ -0,0 +1,121 @@
+<!doctype html>
+<html>
+<head>
+  <title>HTML-AAM img Element Role Verification Test</title>
+  <script src="/resources/testharness.js"></script>
+  <script src="/resources/testharnessreport.js"></script>
+  <script src="/resources/testdriver.js"></script>
+  <script src="/resources/testdriver-vendor.js"></script>
+  <script src="/resources/testdriver-actions.js"></script>
+  <script src="/wai-aria/scripts/aria-utils.js"></script>
+</head>
+<body>
+
+
+<p>Tentative test for the expected role mappings of the <code>img</code> element to ensure
+   it is exposed with the correct role when the <code>src</code>, <code>srcset</code>, both or neither
+   attributes are specified.</p>
+  <p>The computedrole mappings are defined in <a href="https://w3c.github.io/html-aam/">HTML-AAM</a>.<p>
+
+<p>Merge the outcome of this test into roles.html in the appropriate alphabetical order, when it is no longer considered tentative:</p>
+
+<div hidden id=labelledby>x</div>
+
+<h2>Image role:</h2>
+<img data-testname="el-img-no-src-or-srcset-but-name-from-alt" data-expectedrole="image" class="ex"
+  alt=x>
+<img data-testname="el-img-no-src-or-srcset-but-name-from-title" data-expectedrole="image" class="ex"
+  title=x>
+<img data-testname="el-img-no-src-or-srcset-but-name-from-aria-label" data-expectedrole="image" class="ex"
+  aria-label=foo>
+<img data-testname="el-img-no-src-or-srcset-but-name-from-aria-labelledby" data-expectedrole="image" class="ex"
+  aria-labelledby=labelledby>
+
+<img data-testname="el-img-srcset-no-src-no-name" data-expectedrole="image" class="ex"
+  srcset="data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==">
+<img data-testname="el-img-srcset-and-empty-src-no-name" data-expectedrole="image" class="ex"
+  srcset="data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==" src>
+<img data-testname="el-img-srcset-broken-value-and-empty-src-no-name" data-expectedrole="image" class="ex"
+  src srcset="...">
+<img data-testname="el-img-src-broken-value-and-empty-srcset-no-name" data-expectedrole="image" class="ex"
+  srcset src="...">
+
+<img data-testname="el-img-empty-src-with-name-from-title" data-expectedrole="image" class="ex"
+  src title=x>
+<img srcset data-testname="el-img-empty-srcset-with-name-from-title" data-expectedrole="image" class="ex"
+  srcset title=x>
```

Comment:
Duplicate srcset attributes on this and below lines

Reply:
Not sure I understand the usage here. I assumed @scottaohara was testing the empty attr and I don't see the duplication. Maybe it's been resolved? Also, is 'image` the best expected role for an empty `srcset` with no other `src`?